recipes/fs_filepicker/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "fs_filepicker" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "45108b6fe9fbbcd434bee4125ae7901f4976fcb7b2ce3a480075d778c4261932" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ReimarBauer/fs_filepicker/archive/0.0.1.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
it looks like you need the `past` library in the `setup.py`, which is not currently available through the `conda-forge` or `defaults` channels. You'll need to add a recipe for that first before you can build this recipe.

Reply:
ok, thx for this hint. 
I do :)

Reply:
@synapticarbors past should be there because it is in the future package.

https://pypi.python.org/pypi/past/0.11.1
To install past, use: 
$ pip install future

The mistake is not to use it in build requirements